### DISCUSSION TITLE
Use Excalidraw's library feature

### DISCUSSION THREAD

**Discussion by andrewweston** [GitbookIO/community#99] [Feature Requests]
### What's your feature/idea?

I would like to use Excalidraw's library feature to import a public library as well as create my own reusable diagram components.

### What's your desired outcome?

Reusable diagramming components

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by aikoii** [comment]
Hi @andrewweston!

You can already import drawings with .excalidraw format into GitBook 🙂. 

Here is how you can do it: https://github.com/GitbookIO/community/discussions/23#discussioncomment-4708949
